src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -66,6 +66,9 @@ public static FormattableString GetIdentifiersFormattable(this IEnumerable<strin
                 _ => FormattableStringFactory.Create(GetNamesForMethodCallFormat(count, 'I'), identifiers.ToArray<object>())
             };
 
+        public static FormattableString GetParameterReferenceExpressions(this IReadOnlyCollection<Parameter> parameters)
+            => GetIdentifiersFormattable(parameters.Select(p => p.IsPropertyBag ? $"options.{p.Name.FirstCharToUpperCase()}" : p.Name), parameters.Count);
```

Comment:
since we get the name of our parameters by calling `ToVariableName`, we should not just simply call FirstCharToUpperCase, because `ToVariableName` might do something more especially we are in a common place not specific to DPG.
Could we change this to `ToCleanName` instead? This is opposite operation to `ToVariableName` I believe.

Reply:
Changed to `ToCleanName`